source/quickstart/container.rst

```diff
@@ -6,35 +6,356 @@ Quickstart for Containers
+         
+            The example above works this way:
+   
+            - The ``mkdir`` command creates a new local directory at ``~/minio/data`` in your home directory.
+            - The ``docker run`` command starts the MinIO container with the ``-v`` argument to map the local path (``~/minio/data``) to the specified virtual container directory (``/data``).
+            - ``-p`` binds a local port to a container port.
+            - ``-v`` sets a file path as a persistent volume location for the container to use.'
+              When MinIO writes data to ``/data``, that data mirrors to the local path ``~/minio/data``, allowing it to persist between container restarts.
+              You can replace ``~/minio/data`` with another local file location to which the user has read, write, and delete access.
+            - ``-e`` sets the environment variables :envvar:`MINIO_ROOT_USER` and :envvar:`MINIO_ROOT_PASSWORD`, respectively.
+              These set the :ref:`root user credentials <minio-users-root>`.
+              Change the example values to use for your container.
+                 
+         .. tab-item:: Windows
+   
+            .. code-block:: shell
+               :class: copyable
+   
+               docker run \
+                  -p 9000:9000 \
+                  -p 9001:9001 \
+                  --name minio1 \
+                  -v D:\data:/data \
+                  -e "MINIO_ROOT_USER=ROOTUSER" \
+                  -e "MINIO_ROOT_PASSWORD=CHANGEME123" \
+                  quay.io/minio/minio server /data --console-address ":9001"
+               
+            The example above works this way:
+   
+            - The ``docker run`` command starts the MinIO container with the ``-v`` argument to map the local path (``D:\data``) to the specified virtual container directory (``/data``).
+            - ``-p`` binds a local port to a container port.
+            - ``-v`` sets a file path as a persistent volume location for the container to use.'
+              When MinIO writes data to ``/data``, that data mirrors to the local path ``D:\data``, allowing it to persist between container restarts.
+              You can replace ``D:\data`` with another local file location to which the user has read, write, and delete access.
+            - ``-e`` sets the environment variables :envvar:`MINIO_ROOT_USER` and :envvar:`MINIO_ROOT_PASSWORD`, respectively.
+              These set the :ref:`root user credentials <minio-users-root>`.
+              Change the example values to use for your container.
+            
+   .. dropdown:: Docker (Rootless)
+      :name: docker-rootless
+   
+      .. tab-set::
+   
+         .. tab-item:: GNU/Linux or MacOS
+   
+            .. code-block:: shell
+               :class: copyable
+   
+               mkdir -p ${HOME}/data
+   
+               docker run \
+                  -p 9000:9000 \
+                  -p 9001:9001 \
+                  --user $(id -u):$(id -g) \
```

Comment:
IMO we should call this out in the bullet list as to why it's necessary (I'm guessing it just tells Docker to run as the current user/group?)